packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec

```diff
@@ -90,20 +93,16 @@ mv %{buildroot}%{gem_instdir}/bundler.d/%{plugin_name}.rb \
 
 # sample config
 mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
-mv %{buildroot}%{gem_instdir}/settings.d/pulp.yml.example \
-   %{buildroot}%{foreman_proxy_settingsd_dir}/pulp.yml
 mv %{buildroot}%{gem_instdir}/settings.d/pulpcore.yml.example \
    %{buildroot}%{foreman_proxy_settingsd_dir}/pulpcore.yml
-mv %{buildroot}%{gem_instdir}/settings.d/pulpnode.yml.example \
-   %{buildroot}%{foreman_proxy_settingsd_dir}/pulpnode.yml
 
 %files
 %dir %{gem_instdir}
-%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/pulp.yml
 %config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/pulpcore.yml
-%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/pulpnode.yml
 %license %{gem_instdir}/LICENSE
+%{gem_instdir}/bundler.d
 %{gem_libdir}
+%{gem_instdir}/settings.d
```

Comment:
Why is this directory needed? There should be no `settings.d` directory in `%{gem_instdir}`

Reply:
Good question, gemr2rpm added these from the template.

Reply:
Ahh, cause they exist in the gem itself and are packaged inside the gem e.g. https://github.com/theforeman/smart_proxy_pulp/tree/master/settings.d 

gem2rpm automatically added them